.github/workflows/ci-tests-store.yml

```diff
@@ -10,6 +10,11 @@ on:
     paths:
       - ".actions/*"
       - "requirements/store/**"
+      - "src/lightning/__about__.py"
+      - "src/lightning/__init__.py"
+      - "src/lightning/__main__.py"
+      - "src/lightning/__setup__.py"
+      - "src/lightning/__version__.py"
```

Comment:
These changes should be reflected in checkgroup

Reply:
They are taken from checkgroup :)